packages/components/tests/dummy/app/templates/components/tabs.hbs

```diff
@@ -5,8 +5,15 @@
 <section>
   <h3 class="dummy-h3" id="overview"><a href="#overview" class="dummy-link-section">§</a> Overview</h3>
   <p class="dummy-paragraph">
-    Tabs allow users to move between different views within the same context and at the same level of hierarchy, ie. UI
-    vs CLI, macOS vs Windows vs Linux, etc.
+    Tabs allow users to
+    <strong>move between different views within the same context</strong>
+    and at the same level of hierarchy, ie. UI vs. CLI, macOS vs. Windows vs. Linux, etc.
+  </p>
+  <p class="dummy-paragraph">
+    Tabs are not intended for primary navigation. For this reason, we chose to use HTML
+    <code class="dummy-code">button</code>
+    elements for activating individual tabs vs. links. If your use case includes a usage of a Tabs-like component for
+    primary navigation, consider reaching out to your designer and working with them to refactor the design if feasible.
```

Comment:
Do you think we should add also a note about the fact that, if they need to store the active tab in the URL for persistency, they should use the `@onClickTab` handler and do it themselves? Maybe where we show the example _"Pass in a function that gets called when a tab is clicked"_ we could say that this is the suggest way to do if they need to have this functionality?

Reply:
Ok, let me see...

Reply:
I added new language, please see what you think. (A brief sentence in the Overview and further explanation in the "How to use" section.) https://hds-components-git-hds-1118-update-tabs-docs-hashicorp.vercel.app/components/tabs

Reply:
Thanks